app/views/items/_item_smallscreen.html.erb

```diff
@@ -50,7 +50,7 @@
   <%# <%= button_to "Destroy this item", @item, method: :delete %>
   <div class="button-blocker">
   </div>
-  <%= if current_user.lending_rights?(@item)
+  <%= if current_user.lending_rights?(@item) && @src_is_qrcode
```

Comment:
Könnt ihr hier anstatt dem `&&` ein `and` nutzen? Zumindest in unserer Codebase wurde bisher recht konsistent die ausgeschriebene Variante benutzt.